OutOfSchool/OutOfSchool.WebApi/Services/ChildService.cs

```diff
@@ -160,6 +164,49 @@ public async Task<SearchResult<ChildDto>> GetByUserId(string userId, OffsetFilte
             return searchResult;
         }
 
+        /// <summary>
+        /// Get children by WorkshopId.
+        /// </summary>
+        /// <param name="workshopId">ParentId.</param>
+        /// <param name="offsetFilter">Filter to get a part of all children that were found.</param>
+        /// <returns>A <see cref="Task{TResult}"/> representing the result of the asynchronous operation.
+        /// The result is a <see cref="SearchResult{ChildDto}"/> that contains the count of all found children and a list of children that were received.</returns>
+        /// <exception cref="ArgumentNullException">If one of the parameters was null.</exception>
+        /// <exception cref="ArgumentException">If one of the parameters was not valid.</exception>
+        /// <exception cref="SqlException">If the database cannot execute the query.</exception>
+        public async Task<SearchResult<ChildDto>> GetApprovedByWorkshopId(Guid workshopId, OffsetFilter offsetFilter)
+        {
+            ValidateOffsetFilter(offsetFilter);
+
+            logger.LogDebug($"Getting Children by WorkshopId: {workshopId} started. Amount of children to take: {offsetFilter.Size}, skip first: {offsetFilter.From}.");
+
+            var applications = await applicationRepository.GetByFilter(p => p.WorkshopId == workshopId && p.Status == ApplicationStatus.Approved).ConfigureAwait(false);
+            HashSet<Guid> childrenGuids = new HashSet<Guid>();
```

Comment:
As we are not payed for code lines count next code 
```
HashSet<Guid> childrenGuids = new HashSet<Guid>();
foreach (var app in applications)
{
    childrenGuids.Add(app.ChildId);
}
```
could (I believe) be rewritten into
```
var childrenGuids = new HashSet<Guid>(applications.Select(app => app.ChildId));
```